backend/docker-compose.yml

```diff
@@ -6,35 +6,38 @@ services:
       context: .
       target: demo
       args:
-        SECRET_KEY: ${SECRET_KEY:?}
+        SECRET_KEY: tmp_key
```

Comment:
Replace with ${SECRET_KEY:-tmp_key}

Which uses tmp_key as default if SECRET_KEY is not set